tests/Http/ActionTest.php

```diff
@@ -33,7 +33,7 @@ public function testCreatingAnAction()
             'anonymous' => false,
             'noun' => 'things',
             'verb' => 'done',
-        ]);
+        ])->assertRedirect('/actions/1');
```

Comment:
This worked locally, but the DB in [CircleCI](https://circleci.com/gh/DoSomething/rogue/155?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) had an ID of 20. Same happened for campaigns (was 9067 instead of 9000)